portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,39 @@
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+"use strict";
+let popupContentMap = new Map();
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+let PHILOSOPHY_STR_CONTENT = "Right now, I'm reading  'On the Aesthetic Education of Man' by Friedrich Schiller. It's where I found the quote at the top of the page.";
+let BOARD_GAMES_STR_CONTENT = "In quarantine, I've been playing Betrayal and Catan recently, which are both table-top games. My favorite casual games are Codenames and Blokus - both have online versions, so if you ever want to play a game, let me know!";
+let MOVIES_STR_CONTENT = "I have been on a horror movie kick recently - I just watched Hush, The Ring, and Netflix's The Platform";
+let TV_SHOWS_STR_CONTENT = "I enjoy Parks and Rec, M*A*S*H, King of the Hill, and I recently started watching Aziz Ansari's Master of None";
+let MUSIC_STR_CONTENT = "Top 5: Dunno - Mac Miller, Summerhouse - Kota the Friend, Fishing for Fishies - King Gizzard, Volcano - Jimmy Buffett, Woods - Bon Iver";
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let DRONES_STR_CONTENT = "On campus, I am a research assistant to a PhD student working on drone swarms. During the school year, I work about 10-15 hours a week in the lab coding, gathering data, or helping with data visualization. I am also a member of my school's UAV robotics club, where I work mostly with control systems";
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+let CODING_STR_CONTENT = "I have taken two college classes in Java about algorithms and data structures. In my work as a research assistant, I code in C++ and python.";
+let WEB_DEV_STR_CONTENT = "And, as part of this project, I learned HTML, CSS and Javascript";
+let MACHINE_LEARNING_STR_CONTENT = "Recently, I have been learning tensorflow to apply it to my research project.";
+
+popupContentMap.set("philosophy", PHILOSOPHY_STR_CONTENT);
+popupContentMap.set("boardGames", BOARD_GAMES_STR_CONTENT);
+popupContentMap.set("movies", MOVIES_STR_CONTENT);
+popupContentMap.set("tvShows", TV_SHOWS_STR_CONTENT);
+popupContentMap.set("music", MUSIC_STR_CONTENT);
+
+popupContentMap.set("drones", DRONES_STR_CONTENT);
+
+popupContentMap.set("coding", CODING_STR_CONTENT);
```

Comment:
Inconsistent new lines here. So get rid of new lines?

Reply:
Fixed